src/app/screens/speedUpTransaction/index.tsx

```diff
@@ -127,11 +127,20 @@ function SpeedUpTransactionScreen() {
     } finally {
       setIsLoading(false);
     }
-  };
+  }, [
+    selectedAccount,
+    id,
+    transaction,
+    accountType,
+    network.type,
+    seedVault,
+    btcClient,
+    isLoading,
+  ]);
 
   useEffect(() => {
     fetchRbfData();
-  }, [selectedAccount, id, transaction]);
+  }, [selectedAccount, id, transaction, fetchRbfData]);
```

Comment:
no need for `selectedAccount, id, transaction` here. Those are covered by fetchRbfData now 👍